public/vendor/plugins/tribute/tribute.css

```diff
@@ -26,6 +26,7 @@
       display: inline-block;
       vertical-align: middle;
       width: 28px;
+      height: 28px;
```

Comment:
Vendor css should not be changed. Instead create new rule in gitea style less file

Reply:
The vendor css has apparently already been altered, if you take a look at the original repo:
https://github.com/zurb/tribute/blob/master/dist/tribute.css

Then we should reset it to default and transport the changes been done to the gitea style less file as well.